lib/pretty_text.rb

```diff
@@ -324,8 +324,11 @@ def self.extract_links(html)
     links = []
     doc = Nokogiri::HTML5.fragment(html)
 
-    # remove href inside quotes & elided part
-    doc.css("aside.quote a, .elided a").each { |a| a["href"] = "" }
+    # extract onebox links
+    doc.css("aside.onebox").each { |onebox| links << DetectedLink.new(onebox["data-onebox-src"], false) }
```

Comment:
You can do this to ensure the onebox has a `data-onebox-src` attribute.

```suggestion
    doc.css("aside.onebox[data-onebox-src]").each { |onebox| links << DetectedLink.new(onebox["data-onebox-src"], false) }
```